src/nice.bootstrap/navigation/_nav-tabs.scss

```diff
@@ -3,7 +3,7 @@
 // E.g. https://www.nice.org.uk/about/what-we-do/into-practice/education-learning-and-professional-development/case-scenarios
```

Comment:
This page doesn't exist any more, maybe use https://www.nice.org.uk/about/what-we-do/into-practice/audit-and-service-improvement/assessment-tools instead?